src/relay/op/tensor/transform.cc

```diff
@@ -254,6 +254,12 @@ bool ConcatenateRel(const Array<Type>& types,
     if (e_dtype != dtype) {
       throw relay::Error("relay.concatenate requires all tensors have the same dtype");
     }
+    for (int j = 0; j < first->shape.size(); ++j) {
```

Comment:
s/int/size_t

Reply:
Sure, I would like to fix it. Some questions
1. Are there any special points with int / size_t? 
2. If, shape.size() is downcased to int, is size_t still preferred?

Reply:
This is because `size()` returns `size_type` which is implementation defined

https://en.cppreference.com/w/cpp/types/size_t

Reply:
Fixed. Can you have a look?